context/app/static/js/components/detailPage/visualization/VisualizationWrapper/VisualizationError.jsx

```diff
@@ -1,25 +1,26 @@
 import React, { useCallback } from 'react';
 import { ErrorBoundary as SentryErrorBoundary } from '@sentry/react';
+import { faro } from '@grafana/faro-web-sdk';
 
 import { DetailPageSection } from 'js/components/detailPage/style';
 import { SpacedSectionButtonRow } from 'js/shared-styles/sections/SectionButtonRow';
+import DetailsAccordion from 'js/shared-styles/accordions/DetailsAccordion';
 import { StyledSectionHeader } from '../Visualization/style';
 import { VisualizationErrorBoundaryBackground } from './style';
 
 function VisualizationFallback({ isPublicationPage, uuid, shouldDisplayHeader }) {
   return useCallback(
-    ({ componentStack }) => {
+    ({ message, componentStack }) => {
       return (
         <DetailPageSection id={isPublicationPage ? `visualization-${uuid}` : 'visualization'}>
           <SpacedSectionButtonRow
             leftText={shouldDisplayHeader ? <StyledSectionHeader>Visualization</StyledSectionHeader> : undefined}
           />
           <VisualizationErrorBoundaryBackground>
             <div>The Vitessce visualization encountered an error. Please try again or contact support.</div>
-            <details>
-              <summary>Click to expand error details</summary>
-              <div>{componentStack}</div>
-            </details>
+            <DetailsAccordion summary="Click to expand error details">
+              {message} {componentStack}
+            </DetailsAccordion>
```

Comment:
I switched this to use the new details-like accordion component we created for the data products section. I also added the error message itself, since that was previously getting lost.